ui/raidboss/data/03-hw/trial/sephirot-ex.txt

```diff
@@ -1,234 +1,155 @@
+1143.2 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1143.8 "Life Force" sync /:Sephirot:157A:/
+1143.8 "Spirit" sync /:Sephirot:157B:/
+1149.4 "Malkuth" sync /:Sephirot:1582:/
+1150.5 "Adds Spawn" sync /03:........:Added new combatant Storm Of Words/
+1158.7 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1159.3 "Spirit" sync /:Sephirot:157B:/
+1159.3 "Life Force" sync /:Sephirot:157A:/
+1164.9 "Fiendish Wail" sync /:Sephirot:1576:/
+1166.5 "Yesod" sync /:Sephirot:157E:/
+1172.2 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1172.8 "Life Force" sync /:Sephirot:157A:/
+1172.8 "Spirit" sync /:Sephirot:157B:/
+1188.4 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1189.0 "Life Force" sync /:Sephirot:157A:/
+1189.0 "Spirit" sync /:Sephirot:157B:/
+1191.6 "Pillar of Severity" sync /:Sephirot:1585:/
+1195.3 "Impact of Hod" sync /:Sephirot:172C:/
+1199.7 "Ascension" sync /:Coronal Wind:1584:/
+
+1207.6 "Yesod" sync /:Sephirot:157E:/ window 30,30
+1213.5 "Force Field" sync /:Sephirot:1587:/
+1225.7 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1226.3 "Life Force" sync /:Sephirot:157A:/
+1226.3 "Spirit" sync /:Sephirot:157B:/
+1231.9 "Fiendish Wail" sync /:Sephirot:1576:/
+1240.0 "Da'at Tethers" # sync /:Sephirot:1574:/ duration 3.2
+1250.6 "Fiendish Wail" sync /:Sephirot:1576:/
+1257.6 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1258.2 "Life Force" sync /:Sephirot:157A:/
+1258.2 "Spirit" sync /:Sephirot:157B:/
+1265.7 "Earth Shaker" sync /:Sephirot:157D:/
+1267.4 "Yesod" sync /:Sephirot:157E:/
+1275.0 "Da'at spread" # sync /:Sephirot:1573:/
+1282.4 "Fiendish Wail" sync /:Sephirot:1576:/
+1289.5 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1290.2 "Spirit" sync /:Sephirot:157B:/
+1298.3 "Pillar of Mercy" sync /:Sephirot:1580:/
+1298.6 "Yesod" sync /:Sephirot:157E:/
+1303.5 "Pillar of Mercy" sync /:Sephirot:1580:/
+1307.4 "Pillar of Mercy" sync /:Sephirot:1580:/
+1316.6 "Earth Shaker" sync /:Sephirot:157D:/
+1326.1 "Da'at spread" # sync /:Sephirot:1573:/
+1327.0 "Yesod" sync /:Sephirot:157E:/
+1333.4 "Fiendish Wail" sync /:Sephirot:1576:/
+1340.5 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1341.1 "Life Force" sync /:Sephirot:157A:/
+1341.1 "Spirit" sync /:Sephirot:157B:/
+1346.7 "Malkuth" sync /:Sephirot:1582:/
+1347.8 "Adds Spawn" sync /03:........:Added new combatant Storm Of Words/
+1356.0 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1356.6 "Spirit" sync /:Sephirot:157B:/
+1356.6 "Life Force" sync /:Sephirot:157A:/
+1362.2 "Fiendish Wail" sync /:Sephirot:1576:/
+1363.8 "Yesod" sync /:Sephirot:157E:/
+1369.5 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1370.1 "Life Force" sync /:Sephirot:157A:/
+1370.1 "Spirit" sync /:Sephirot:157B:/
+1385.7 "Chesed Gevurah" sync /:Sephirot:157[89]:/
+1386.3 "Life Force" sync /:Sephirot:157A:/
+1386.3 "Spirit" sync /:Sephirot:157B:/
+1388.9 "Pillar of Severity" sync /:Sephirot:1585:/
+1392.6 "Impact of Hod" sync /:Sephirot:172C:/
+1397.0 "Ascension" sync /:Coronal Wind:1584:/
+1403.9 "Pillar of Severity Enrage" sync /:Sephirot:1585:/
```

Comment:
Yeah, I think my memory is that this is a "rotational" hard enrage.  I don't think I've ever had a slow enough group in phase 1 to know if there's a time-based one.